pgjdbc/src/main/java/software/aws/rds/jdbc/postgresql/Driver.java

```diff
@@ -67,12 +66,10 @@
  * @see java.sql.Driver
  */
 public class Driver extends org.postgresql.Driver {
```

Comment:
Let's make register() and deregister() methods synchronized similar to community Driver.